### PR TITLE
Help converter foss tyler

### PR DESCRIPTION
**Pull request by tylerslijboom-r7**
## Proposed Changes

Describe the proposed changes:

help.md and plugins.spec.yaml updates for plugins starting with the letter i
  -

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `./run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `./run -T tests/example.json -d -j`
    * All actions/triggers shortcut `./run -T all -d -j` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `./run -R tests/example.json -d -j`
    * All actions/triggers shortcut `./run -R all -d -j` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `./run -A -R all -T all`. For single action validation: `./run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`./run ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

File: ibm_resilient_incident/plugin.spec.yaml [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 version: 1.0.0
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/ibm_resilient_incident
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.ibm.com/security/intelligent-orchestration/resilient
 tags:
-- resilient
-- orchestration
-- incident response
+
```
Comment (lmilby-r7) [inline ibm_resilient_incident/plugin.spec.yaml]: remove extra blank line

File: ifconfig_co/plugin.spec.yaml [hunk 2]
```diff
@@ -11,10 +11,12 @@ version: 1.0.1
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/ifconfig_co
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/ipconfig
 tags:
-- ip
-- nat
-- public
+
```
Comment (lmilby-r7) [inline ifconfig_co/plugin.spec.yaml]: remove blank lines

File: influxdb/plugin.spec.yaml [hunk 3]
```diff
@@ -11,11 +11,12 @@ version: 1.0.0
...
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://docs.influxdata.com/influxdb
 tags:
-- database
-- influxdb
+  - database
 hub_tags:
-  use_cases: []
+  use_cases: [ reporting_and_analytics ]
+
```
Comment (lmilby-r7) [inline influxdb/plugin.spec.yaml]: remove blank line

File: influxdb/help.md [hunk 4]
```diff
@@ -1,7 +1,16 @@
 
 # InfluxDB
 
-## About
+# Key Features
+
+* Metrics
```
Comment (jschipp-r7) [inline influxdb/help.md]: Store data in InfluxDB
Retrieve data from InfluxDB

File: influxdb/plugin.spec.yaml [hunk 5]
```diff
@@ -11,11 +11,12 @@ version: 1.0.0
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/influxdb
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://docs.influxdata.com/influxdb
```
Comment (jschipp-r7) [inline influxdb/plugin.spec.yaml]: Use their main product URL

File: ipify/help.md [hunk 6]
```diff
@@ -4,14 +4,11 @@
 
 # Key Features
 
-* Feature 1
-* Feature 2
-* Feature 3
+* IP Lookup of a Domain
```
Comment (jschipp-r7) [inline ipify/help.md]: Obtain your public IP address

File: ipinfo/help.md [hunk 7]
```diff
@@ -3,16 +3,13 @@
 [IPinfo](https://ipinfo.io/) returns IP information including: hostname, ASN, organization, and GeoIP data.
 The IPinfo plugin queries the IPinfo webpage via its API and returns relevant information.
 
-# Key Features
+# Features
```
Comment (jschipp-r7) [inline ipinfo/help.md]: Should be Key Features

```suggestion
# Key Features
```

File: ipinfo/help.md [hunk 8]
```diff
@@ -3,16 +3,13 @@
...
-* Feature 1
-* Feature 2
-* Feature 3
+* IP Info, IP or domain lookup
 
 # Requirements
 
-* Example: Requires an API Key from the product
-* Example: API must be enabled on the Settings page in the product
+* API token from the vendor is needed
```
Comment (jschipp-r7) [inline ipinfo/help.md]: Use the example as a style convention
```suggestion
* Requires an API Key from IPInfo
```

File: ipintel/help.md [hunk 9]
```diff
@@ -4,14 +4,11 @@
 
 # Key Features
 
-* Feature 1
-* Feature 2
-* Feature 3
+* Detect Users locations
```
Comment (jschipp-r7) [inline ipintel/help.md]:
```suggestion
* IP enrichment
```

**Pull request merged**